src/components/molecules/timeline-events/order-edit/declined.tsx

```diff
@@ -1,25 +1,44 @@
-import { useAdminCustomer } from "medusa-react"
+import { useAdminCustomer, useAdminOrderEdit, useAdminUser } from "medusa-react"
 import React from "react"
-import { ByLine } from "."
+
 import { OrderEditEvent } from "../../../../hooks/use-build-timeline"
 import XCircleIcon from "../../../fundamentals/icons/x-circle-icon"
 import EventContainer from "../event-container"
+import { ByLine } from "."
 
 type EditDeclinedProps = {
   event: OrderEditEvent
 }
 
 const EditDeclined: React.FC<EditDeclinedProps> = ({ event }) => {
-  const { customer } = useAdminCustomer(event.edit.declined_by as string)
+  const { order_edit: orderEdit } = useAdminOrderEdit(event.edit.id)
+
+  const declinedByAdmin = event.edit.declined_by?.startsWith("usr")
```

Comment:
suggestion: I ve seen that check twice. Should we move "usr" or "cus" outside of here as const util for the order edit events to be used when needed?

Reply:
where is the second check, [this one](https://github.com/medusajs/admin/pull/844/files#diff-c09938aa0b5e0acfc98f5670a62a66485da4ba9de87349c0afff166ec94579f9R14)?

Reply:
yep, just to ensure that there is no type when someone add a new check like this one or if we change the value